.gitmodules

```diff
@@ -3,7 +3,8 @@
 	url = https://github.com/NCAR/TEMPO.git
 [submodule "src/core_atmosphere/physics/UGWP"]
 	path = src/core_atmosphere/physics/UGWP
-	url = https://github.com/mdtoyNOAA/UGWP.git
+	url = https://github.com/NOAA-GSL/UGWP.git
+        branch = main_knob_ugwp_tauamp_namelist
```

Comment:
I think the standard practice would be to point to your primary branch (like TEMPO and MYNN) or a release/tag. Is there a reason to point to this somewhat confusing branch name?